docs/api.md

```diff
@@ -665,6 +665,23 @@ Create and return an instance of the class bot.
 
 A sync representation of the world. Check the doc at http://github.com/PrismarineJS/prismarine-world
 
+#### Events:
```

Comment:
not sure if we should duplicate pworld doc here, but I guess let's let it there for now